File: src/main/java/com/pingsu/biggerlecternterminal/Biggerlecternterminal.java
package com.pingsu.biggerlecternterminal;

import com.mojang.logging.LogUtils;
import net.minecraftforge.fml.common.Mod;
import org.slf4j.Logger;

@Mod(Biggerlecternterminal.MODID)
public class Biggerlecternterminal {
    public static final String MODID = "biggerlecternterminal";

    private static final Logger LOGGER = LogUtils.getLogger();

    public Biggerlecternterminal() {
        LOGGER.info("Loaded {}", MODID);
    }
}

File: src/main/kotlin/com/pingsu/biggerlecternterminal/Biggerlecternterminal.kt
package com.pingsu.biggerlecternterminal

import net.neoforged.fml.common.Mod
import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

@Mod(Biggerlecternterminal.ID)
object Biggerlecternterminal {
    const val ID = "biggerlecternterminal"

    val LOGGER: Logger = LogManager.getLogger(ID)

    init {
        LOGGER.info("Loaded {}", ID)
    }
}
